app/javascript/components/forms/components/radio/component.jsx

```diff
@@ -54,6 +64,17 @@ class Radio extends PureComponent {
                       <label className="radio-label" htmlFor={id}>
                         <span />
                         {option.label}
+                        {otherRegex.test(option.label) &&
+                          selectedOption &&
+                          otherRegex.test(selectedOption) && (
+                          <Field
```

Comment:
Im not a big fan of including this inside this component. It feels like a very specific thing to the profile form. Surely just a conditional field element below in the form is simple enough? Also the styles of the input you have included are not consistent with any other input field. It should be below and the same height etc.

<img width="391" alt="Screenshot 2020-02-04 at 16 15 23" src="https://user-images.githubusercontent.com/20288774/73757488-9549c780-4769-11ea-95c2-3e553708a5bb.png">

Reply:
it's half the height on purpose, otherwise it moves the "other" option too far from the others. It was a decision not very consistent with the UI kit, but in order to respect the parent component first. 

It could be a conditional field, yes, passed as a prop. I did this for any fields having "other" and "write" keywords, as it sounds like a common case for radio buttons, but it could be an extra field in the option object, agree (less specific that way).

Reply:
In this case we need to ask the designers.